### PR TITLE
Scheduled post left gm fix

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixed a bug where GM member count shows up as 0 in scheduled posts

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-61267

#### Release Note
```release-note
None
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
More `DataLoader` uses <3

File: webapp/channels/src/components/drafts/draft_title/draft_title.test.tsx [hunk 1]
```diff
@@ -10,31 +10,36 @@ import type {UserProfile} from '@mattermost/types/users';
...
         teammateId: '',
-        type: '' as 'channel' | 'thread',
+        type: 'channel' as 'channel' | 'thread',
     };
 
     it('should match snapshot', () => {
         const store = mockStore();
 
-        const wrapper = shallow(
+        const {container} = render(wrapIntl(
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/drafts/draft_title/draft_title.test.tsx:35]: you dont need to introduce another util if you use renderWithContext directly.

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

File: webapp/channels/src/components/drafts/draft_title/draft_title.test.tsx (hunk 1 above)
Comment (harshilsharma63) [reply webapp/channels/src/components/drafts/draft_title/draft_title.test.tsx:35]: Ah thanks! Done.

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**